### PR TITLE
Oembed feature for Rocket.Chat 

### PR DESCRIPTION
**Pull request by CulturalProfessor**
Fixes (#45 )
image

### CODE REVIEW

File: imports/models/RocketChatCollection.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    if (ifBoardExists == undefined) {
+      try {
+        await this.collection.insertAsync(board)
+        return 'success' 
+      } catch (error) {
+        console.error('Error inserting board:', error)
+        return 'error'
+      }
+    } else {
+      console.log(ifBoardExists._id)
```
Comment (shiqimei) [inline imports/models/RocketChatCollection.ts:32]: remove logs

File: imports/models/RocketChatCollection.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    return result
+  }
+
+  public async getRocketChatBoard( 
+    userId: string,
+    username: string,
+    boardname: string
+  ): Promise<RocketChatUserRecord | null> {
+    const document = await this.collection.findOneAsync({ _id: boardname, userId: userId, username: username })
+    console.log('Document', document)
```
Comment (shiqimei) [inline imports/models/RocketChatCollection.ts:53]: remove logs

**Comment by shiqimei** [comment]
Thanks for you contribution, @CulturalProfessor !

**Comment by CulturalProfessor** [comment]
> Thanks for you contribution, @CulturalProfessor !

@shiqimei :pray:

**Pull request merged**